solidity/random-beacon/README.adoc

```diff
@@ -55,39 +55,38 @@ Instead of a v1 ticket-based approach for group selection, we use a sortition
 pool. Group creation start transaction is embedded into relay request
 transaction and locks a sortition pool. From this moment, no operator can enter
 or leave the pool. Once a new relay entry appears on the chain, all off-chain
-clients perform group selection optimistically calling
-`RandomBeacon.selectGroup(seed)` view function for free. Seed is available in
-`DkgStarted` event emitted when the group creation starts. After determining
-group members, clients should perform off-chain distributed key generation (DKG).
- <<dkg-submit-eligibility,Eligible group member>> submits the result to the chain
- calling `RandomBeacon.submitDkgResult(DKG.Result calldata dkgResult)` function.
- Once the result is submitted, a challenge period starts.
+clients perform group selection by calling `RandomBeacon.selectGroup()` view
+function for free. After determining group members, clients should perform
+off-chain distributed key generation (DKG). One of the group members submits
```

Comment:
> One of the group members
Could we link that to the Operator-Only section in Transaction Incentives?

Reply:
1fadd0bedacc1e8332f6ef3c24e1f6db4acc3322